### ISSUE TITLE
Maverick nibbler does not work on U2+ emulated drive.

### ISSUE THREAD

**Issue by Zibri** [GideonZ/1541ultimate#325]
To help a friend dump a disk in G64 we tried doing this:

Device8: real 1541
Device 9: u2+ drive A
Then created an emty G64 and mounted on A
Then we ran Maverick Single Nibbler.
The result was an unreadable disk even if as a test we used a normal formatted disk.

**Comment by Grrrolf** [comment]
Have you tried copying the disk using real target hardware? Does the resulting copy/disk work if you copy it that way?

**Comment by radius75** [comment]
I tried Maverick once and it didn't work.

Finally, I copied two floppy disks with TurboNibbler4
The first g64 created was unreadable on Ultimate but worked in Vice. It was twice the size of a standard g64.

I made copies from this weird g64 to g64 in Vice (also TurboNibbler4) and managed to get a working (Ultimate and Vice) g64 of reasonable size.

Weird but it works for me.

**Comment by radius75** [comment]
Only now I noticed that we are talking about the Single Copy option.
You probably damaged the source floppy irreversibly in this way. Read from 08 and write to 08. If the source floppy was write-protected (sticker), nothing happened.
With the setting as you provided, you should use the Dual Nybbler. What program/game are you trying to back up?
![obraz](https://user-images.githubusercontent.com/1929111/233918345-8cb4cd70-6adc-4a00-8359-21e9c9c79b57.png)

**Comment by Zibri** [comment]
> Have you tried copying the disk using real target hardware? Does the resulting copy/disk work if you copy it that way?

yes, since I just tried to copy a dos formatted emty disk, not a protected game.

**Comment by Zibri** [comment]
> Only now I noticed that we are talking about the Single Copy option. You probably damaged the source floppy irreversibly in this way. Read from 08 and write to 08. If the source floppy was write-protected (sticker), nothing happened.
> 
> -- And if you set for Single Nybbler Dev. number 09 is nothing special happened. Read/Write was only on g64.
> 
> With the setting as you provided, you should use the Dual Nybbler. What program/game are you trying to back up? ![obraz](https://user-images.githubusercontent.com/1929111/233918345-8cb4cd70-6adc-4a00-8359-21e9c9c79b57.png)

1) I used SINGLE NYBBLER to test it.
2) I correctly setup up device 8 (source) as the real 1541 and device 9 (destination) as the emulated 1541 DRIVE A
3) the source was a disk formatted with the classic "open15,8,15,"N:test,t1":close15
4) I expected to find a copy of that empty disk in the g64 mounted on drive A

Instead I got an unreadable disk.
I can try again.. but I doubt I made any mistake.

About the dual nybbler, I haven't tried it, but for debugging purposes I am just saying that the single nybbler (which should work anyways) doesn't.

**Comment by radius75** [comment]
> > Have you tried copying the disk using real target hardware? Does the resulting copy/disk work if you copy it that way?
> 
> yes, since I just tried to copy a dos formatted emty disk, not a protected game.

So, have you tried to read/write using only one floppy?
On the real 1541?
The question is whether this can be done, or do nybbler check what sector they overwrite at all?
Have you tried using two floppy disks?

**Comment by Zibri** [comment]
@radius75 
maverick can indeed copy a normally formatted disk using a single 1541...
it's with emulators and U2+ that it has problems..
and yes.. a normally formatted disk works everywhere...
it's MAVERICK >> U2 emulated drive  that has problems.
obviously it's not maverick fault..
clearly it's the emulation that ios lacking (alsovice has many problems with 1541 emulation)

**Comment by radius75** [comment]
I tried copyprotected g64 to g64 on Maverick
(35track). From DriveA.08 to DriveB.09. The copy was successful.
GCR Nybble Copier / Dual Nybbler
New destination g64 placed in Temp folder. Source g64 on USB stick.
I'll plug in the 1541 soon and check the same copy-protected real floppy.

**Comment by radius75** [comment]
I tried copy real floppy to g64.
From 1541-II.08 to destination g64 DriveB.09 (DriveA disabled). New destination g64 placed in Temp folder
The output g64 is corrupted.

Uploaded both files for comparison:
[compare.zip](https://github.com/GideonZ/1541ultimate/files/11362191/compare.zip)

**Comment by Zibri** [comment]
yep. as I said.. I tried even with a normal disk... real floppy > g64 on U2+ = corrupted.
